RELEASE/scripts/autoscend/quests/level_11.ash

```diff
@@ -3134,6 +3154,12 @@ boolean L11_unlockEd()
 		auto_log_info("Attemping to use Map the Monsters to olfact a Tomb Rat.");
 	}
 	
+	if(auto_haveGreyGoose())
+	{
+		auto_log_info("Bringing the Grey Goose to emit some drones at some rats.");
```

Comment:
See comment about the regular rats

Reply:
Added check to make sure we can get rat kings